webapi/src/main/scala/org/knora/webapi/store/triplestore/impl/TriplestoreServiceLive.scala

```diff
@@ -68,12 +54,19 @@ import org.knora.webapi.util.FileUtil
 
 case class TriplestoreServiceLive(
   triplestoreConfig: Triplestore,
-  queryHttpClient: CloseableHttpClient,
-  targetHost: HttpHost,
+  sttp: SttpBackend[Task, ZioStreams],
 )(implicit val sf: StringFormatter)
     extends TriplestoreService
     with FusekiTriplestore {
 
+  private val targetHost = new HttpHost(triplestoreConfig.host, triplestoreConfig.fuseki.port, "http")
```

Comment:
The `Triplestore` config has a boolean `useHttps` property, shouldn't we use it for the protocol?

Reply:
Is it ever in use?

Reply:
Did you check? If not we might as well remove it.

Reply:
Added.